toptal/picasso
# fix(form-input): forward onFocus prop to the underlying component

[FX-1811]

### Description

The `onFocus` callback of the `Form.Input` is not working. This PR forwards the `onFocus` call following the same existing pattern of `onChange` and `onBlur`.

### How to test

- Edit the code on [Storybook](https://picasso.toptal.net/?path=/story/picasso-forms-form--form#default) to add a `onFocus` callback and make sure it works

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[FX-1811]: https://toptal-core.atlassian.net/browse/FX-1811
